stdlib/@tests/stubtest_allowlists/py313.txt

```diff
@@ -116,6 +116,7 @@ pdb.Pdb.do_exceptions
 pdb.Pdb.interaction
 pdb.Pdb.message
 pdb.Pdb.user_opcode
+# `__replace__` to be special cased in dataclasses
```

Comment:
Let me know if I'm wrong in this assumption. I'm assuming something like `__replace__` just needs to be special cased in MyPy/Pyright for dataclasses?

Reply:
Yes, type checkers will have to synthesize the `__replace__` method.